ui/page/odyseeMembership/view.jsx

```diff
@@ -0,0 +1,613 @@
+/* eslint-disable no-console */
+// @flow
+import React from 'react';
+import moment from 'moment';
+import Page from 'component/page';
+import Spinner from 'component/spinner';
+import { Lbryio } from 'lbryinc';
+import { getStripeEnvironment } from 'util/stripe';
+import * as ICONS from 'constants/icons';
+import * as PAGES from 'constants/pages';
+import * as MODALS from 'constants/modal_types';
+import Card from 'component/common/card';
+import MembershipSplash from 'component/membershipSplash';
+import Button from 'component/button';
+import ChannelSelector from 'component/channelSelector';
+import PremiumBadge from 'component/common/premium-badge';
+import useGetUserMemberships from 'effects/use-get-user-memberships';
+import usePersistedState from 'effects/use-persisted-state';
+
+let stripeEnvironment = getStripeEnvironment();
+
+// const isDev = process.env.NODE_ENV !== 'production';
+const isDev = true;
+
+// odysee channel information since the memberships are only for Odysee
+const odyseeChannelId = '80d2590ad04e36fb1d077a9b9e3a8bba76defdf8';
+const odyseeChannelName = '@odysee';
+
+type Props = {
+  history: { action: string, push: (string) => void, replace: (string) => void },
+  location: { search: string, pathname: string },
+  totalBalance: ?number,
+  openModal: (string, {}) => void,
+  activeChannelClaim: ?ChannelClaim,
+  channels: ?Array<ChannelClaim>,
+  claimsByUri: { [string]: any },
+  doFetchUserMemberships: (claimIdCsv: string) => void,
+  incognito: boolean,
+  updateUserOdyseeMembershipStatus: () => void,
+  user: ?User,
+};
+
+const OdyseeMembershipPage = (props: Props) => {
```

Comment:
There is a missing `key` requirement somewhere in this component. It causes subtle redraw issues.

<img width="510" alt="image" src="https://user-images.githubusercontent.com/64950861/156501809-f07e1c4a-cd44-400f-87da-46d85b9c8184.png">